gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -161,6 +161,7 @@
   public static final String EXTRACT_TABLE_TYPE_KEY = "extract.table.type";
   public static final String EXTRACT_NAMESPACE_NAME_KEY = "extract.namespace";
   public static final String EXTRACT_TABLE_NAME_KEY = "extract.table.name";
+  public static final String EXTRACT_TABLE_NAMES_KEY = "extract.table.names";
```

Comment:
Can you please document what exactly the format of the property is? A comma-separated list of strings (table names)?